README.txt

```diff
@@ -109,6 +109,11 @@ If the recurring opportunity will have the same volunteers each time, we’d rec
 
 == Changelog ==
 
+= 1.4.6 =
+* Adjust how the Volunteer Opportunities block is registered to prevent it from being registered on Volunteer Opportunity posts, the Widgets admin page, and the Widgets area in the customizer.
+* Fixed PHP notices that were being caused by variables being undefined when a new instance of the Volunteer Management Opportunities widget was created.
```

Comment:
How about "Fixed PHP notices caused by undefined variables when a new instance of the Volunteer Management Opportunities widget was created."

Reply:
Updated.